.github/workflows/onPRClose.yml

```diff
@@ -50,13 +50,16 @@ jobs:
   destroy-deployed-branch:
     name: Call Azure Destroy Pipeline
     runs-on: ubuntu-latest
-    needs: get-details
     steps:
+    - name: Set new branch name value
+      run: |
+        echo 'chartName='$( echo ${{ github.head_ref || github.ref_name }} | sed "s/_/-/" | awk '{print tolower($0)}' ) >> "$GITHUB_OUTPUT"
+      id: getChartName
     - name: Azure Pipelines Action
-      if: needs.get-details.outputs.issue-number
+      if: steps.getChartName.outputs.chartName != 'staging'
```

Comment:
Could we also check master? just for the odd chance that are merge master to staging